src/commands/settings.py

```diff
@@ -19,3 +22,44 @@ def handle_hide_cuisine(update, context):
     cuisine_to_hide = parse_callback(update.callback_query.data)[1]
     updated_hidden_cuisine = update_hidden_cuisine(update.effective_chat.id, cuisine_to_hide)
     update.callback_query.edit_message_reply_markup(reply_markup=hidden_cuisine_kb(updated_hidden_cuisine))
+
+
+def handle_add_favorite(update, context):
+    food_to_add = ' '.join(context.args)
+    if food_to_add == '':
+        update.message.reply_text(add_favorite_no_input_msg())
+        return
+
+    food_to_add = normalize(food_to_add)
+    favorites = update_favorite_foods(update.effective_chat.id, food_to_add)
+
+    if favorites is None:
+        update.message.reply_text(add_favorite_already_exists_msg())
+        return
+
+    update.message.reply_text(added_favorites_msg(favorites))
```

Comment:
Could we reduce verbosity with one-liner
`update.message.reply_text(added_favorites_msg(favorites) if favorites is None else add_favorite_already_exists_msg())`